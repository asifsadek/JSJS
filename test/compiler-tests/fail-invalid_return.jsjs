val hazardous_fn = /\() => {
    print("eden hazard");
    val ok = "hmm...";
};
