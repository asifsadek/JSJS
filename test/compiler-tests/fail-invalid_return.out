Type error: Assignment expressions cannot be returned
